runsc/fsgofer/filter/config.go

```diff
@@ -103,6 +103,8 @@ var allowedSyscalls = seccomp.MakeSyscallRules(map[uintptr]seccomp.SyscallRule{
 		seccomp.EqualTo(0),
 	},
 	unix.SYS_GETPID:       seccomp.MatchAll{},
+	unix.SYS_GETUID:       seccomp.MatchAll{},
+	unix.SYS_SETREUID:     seccomp.MatchAll{},
```

Comment:
If we need to set uid-s once, we need to do that before installing seccomp filters.

Reply:
will do!

Reply:
I think we will be setting UID per RPC, so we will end up wanting the seccomp filter.

Reply:
hmm but I also need to restore the uid back to gofer's uid. Without calling `getuid(2)` how do I know which uid to restore to?

Reply:
I think you can save the getuid(2) of the gofer process [before the seccomp filters are installed](https://github.com/google/gvisor/blob/f7875f90fda166bd4e72929a19c5a2f2fb5f17f6/runsc/cmd/gofer.go#L256-L266). You can make that information part of `fsgofer.Config{}`. You can see in fsgofer/lisafs.go, how many handlers can access the config, see [example](https://github.com/google/gvisor/blob/f7875f90fda166bd4e72929a19c5a2f2fb5f17f6/runsc/fsgofer/lisafs.go#L498-L501).

Reply:
thank you!